openc3/lib/openc3/utilities/metric.rb

```diff
@@ -129,9 +129,3 @@ def self.add_update_generator(object)
     end
   end
 end
-
-begin
-  require 'openc3-enterprise/utilities/metric'
-rescue LoadError
-  # Open Source Edition - Do nothing here
-end
```

Comment:
Not sure if this was a future enhancement or what ... there is no `openc3-enterprise/utilities/metric`

Reply:
This exists and should be reverted.